client/js/widgets/button.js

```diff
@@ -376,13 +380,22 @@ export class Button extends Widget {
       }
 
       if(a.func == 'LABEL') {
-        setDefaults(a, { value: 0, mode: 'set' });
+        setDefaults(a, { value: 0, mode: 'set', collection: 'DEFAULT' });
         if([ 'set', 'dec', 'inc' ].indexOf(a.mode) == -1)
           problems.push(`Warning: Mode ${a.mode} will be interpreted as add.`);
-        this.w(a.label, label=> {
-          if (this.p('debug')) console.log(`changing ${a.label} ${a.mode} ${a.value}`)
-          label.setText(a.value, a.mode)
-        });
+        if(a.label !== undefined) a.widget = a.label; // remove this line to deprecate the 'label' property in favor of 'widget'
```

Comment:
Didn't we say no aliases? Shouldn't this do a file version bump instead? (We probably shouldn't wait for the pile rewrite anymore.)

Reply:
Right, I was intending to prepare a file version bump, but I'll better do that as a dedicated PR and just get this PR merged.